pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=1.0.0 <2.0.0'
 
 dependencies:
-  args: '>=0.10.0 <0.13.0'
+  args: '>=0.12.0 <0.14.0'
```

Comment:
I'm assuming we do require at least `0.12.0`? I'd like to keep the constraints as broad as possible.